htdocs/langs/en_US/dict.lang

```diff
@@ -250,7 +250,9 @@ CountryMF=Saint Martin
 
 ##### Civilities #####
 CivilityMME=Mrs.
+CivilityMMEShort=CivilityMMEShort
```

Comment:
This line looks strange. After the = we should have a translated text (so a human readable text).

Reply:
i agree  , this is only used for other languages , i'll change it with the same as Mrs.